src/gui/properties/propertieswidget.h

```diff
@@ -30,6 +30,7 @@
 #define PROPERTIESWIDGET_H
 
 #include <QList>
+#include <QModelIndex>
```

Comment:
Is it really required?

Reply:
Yes, if I use forward declaration then it shows the following error:
```
compiling moc_propertieswidget.cpp
moc_propertieswidget.cpp: In static member function ‘static void PropertiesWidget::qt_static_metacall(QObject*, QMetaObject::Call, int, void**)’:
moc_propertieswidget.cpp:167:44: error: invalid use of incomplete type ‘class QModelIndex’
  167 |         case 7: _t->openDoubleClickedFile((*reinterpret_cast< QModelIndex(*)>(_a[1]))); break;
      |                                           ~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/qt/QtCore/qobject.h:54,
                 from /usr/include/qt/QtWidgets/qwidget.h:45,
                 from /usr/include/qt/QtWidgets/QWidget:1,
                 from ../../src/gui/properties/propertieswidget.h:33,
                 from moc_propertieswidget.cpp:10:
/usr/include/qt/QtCore/qmetatype.h:2089:1: note: forward declaration of ‘class QModelIndex’
 2089 | QT_FOR_EACH_STATIC_CORE_CLASS(QT_FORWARD_DECLARE_STATIC_TYPES_ITER)
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from moc_propertieswidget.cpp:10:
../../src/gui/properties/propertieswidget.h:86:44: note:   initializing argument 1 of ‘void PropertiesWidget::openDoubleClickedFile(QModelIndex) const’
   86 |     void openDoubleClickedFile(QModelIndex index) const;
      |                                ~~~~~~~~~~~~^~~~~
make[1]: *** [Makefile:5154: moc_propertieswidget.o] Error 1

```